Core activities (#45)
RockefellerArchiveCenter/DTeamDocs: core-activities.md

# Core Activities

> ## Implement and Develop Systems

Reviewer: ```suggestion
## Implement, Develop and Decommission Systems
```

> We lead strategy for the selection, implementations, and decommissioning of systems key to archival functions. We evaluate workflows, systems, and processes holistically, with an eye towards removing redundancy and eliminating unnecessary manual intervention so that our colleagues are better able to use their unique skills in acquiring, preserving, and providing access to archival collections. We develop systems that fit the RAC's needs and use cases, as well as work with open source applications and their communities.

Reviewer: ```suggestion
We lead strategy for the selection, implementation, and decommissioning of systems key to archival functions. Evaluating workflows, systems, and processes holistically, we try to removing redundancy and eliminate unnecessary manual intervention so that our colleagues are better able to use their unique skills in acquiring, preserving, and providing access to archival collections. We develop systems that fit the RAC's needs and use cases, and also work with open source applications and their communities.
```

> ## Design for User Experience

Reviewer 2: ```suggestion
## Improve User Experience
```

Author: I personally prefer `Design for` over `Improve`. But I'd like @helrond and @bonniegee thoughts on this as well.

I could be swayed to `Improve`, especially since this is a section @HaSistrunk probably feels strongly about.

Reviewer 2: I was just thinking that "User Experience" itself is neutral. It could be good or bad.

Reviewer: I'm more in favor of "Improve" here - it speaks to the ongoing nature of this work, and I don't think it's all "design", at least not in the sense that folks will think about that word.

Reviewer 3: I'm fine with "improve"--I don't have super strong feelings either way.

> We advocate for and improve the usability of archival systems and processes at the RAC by taking a user-centered approach to design and implementation, striving to improve user experience by engaging users throughout all stages of the systems lifecycle. We perform user research, create and lead user experience tests, and involve stakeholders throughout the design process. Because we recognize that accessibility is integral to usability, we are committed to ensuring that people with disabilities have access to our systems and content.

Reviewer 2: ```suggestion
We advocate for and improve the usability of archival systems and processes at the RAC by taking a user-centered approach to design and implementation. We perform user research, create and lead user experience tests, and involve stakeholders throughout the design process. Because we recognize that accessibility is integral to usability, we are committed to ensuring that people with disabilities have access to our systems and content.
```

Author: @HaSistrunk I wouldn't mind having your thoughts about why you wanted to remove the `striving to improve user experience by engaging users throughout all stages of the systems lifecycle`.

Reviewer 2: Yeah, sorry I should have added comments to these suggestions. I was thinking that this was redundant both in using "improve" again, and because we say that we "involve stakeholders throughout the design process, which feels repetitive to "engaging users throughout all stages...".

> We advocate for and improve the usability of archival systems and processes at the RAC by taking a user-centered approach to design and implementation. We strive to improve user experience by engaging users throughout all stages of the implementation and systems lifecycle. We perform user research, create and lead user experience tests, and involve stakeholders throughout the design process. We recognize that accessibility is integral to usability, and are committed to ensuring that people with disabilities have access to our systems and content.

Reviewer: ```suggestion
We advocate for and improve the usability of archival systems and processes at the RAC by taking a user-centered approach to design and implementation, striving to improve user experience by engaging users throughout all stages of the systems lifecycle. We perform user research, create and lead user experience tests, and involve stakeholders throughout the design process. Because we recognize that accessibility is integral to usability, we are committed to ensuring that people with disabilities have access to our systems and content.
```

## Maintain Technical Infrastructure

> Ongoing maintenance and repair of infrastructure are vital to keeping our systems and processes working smoothly. We identify breakdowns in our systems infrastructure and implement solutions to provide for their continued use. We provide product ownership for systems which support archival functions at the RAC and we upgrade and update existing tools and applications to ensure their stability and desired functionality. We plan for the long term maintenance of systems and workflows that support key archival duties, including discovery, access and use of archival collections, reference services, processing, accessioning, digital preservation, and digitization.

Reviewer: ```suggestion
Ongoing maintenance and repair of infrastructure are vital to keeping our systems and processes working smoothly. We identify breakdowns in our systems infrastructure and implement solutions to provide for their continued use. In addition, we provide product ownership for systems which support archival functions at the RAC.  We plan for the long term maintenance of systems and workflows that support key archival duties - including discovery, access and use of archival collections, reference services, processing, accessioning, digital preservation, and digitization - by managing upgrades and updates for existing tools and applications to ensure their stability and desired functionality.
```

## Develop and Provide Training

> We provide training to empower our colleagues in their use of technology, through formal and informal methods such as one-on-one mentoring, workshops and study groups. We identify gaps in knowledge as well as resources to fill those gaps. We are a resource for expertise ranging from the intricacies of programming languages to broader topics such as digital preservation. We learn from our teaching and build organizational redundancy for expertise; we never want to be the sole owners of knowledge about or experience with digital systems and materials.

Reviewer: ```suggestion
We provide training to empower our colleagues in their use of technology, through a variety of methods - such as one-on-one mentoring, workshops and study groups - and across a range of topics, from the intricacies of programming languages to broader topics such as digital preservation. We identify gaps in knowledge as well as resources to fill those gaps. Our teaching is designed to help us continue to learn and build organizational capacity and redundancy for expertise. We never want to be the sole owners of knowledge about or experience with digital systems and materials.
```
